src/button.test.forms.ts

```diff
@@ -0,0 +1,144 @@
+import { html } from 'lit/static-html.js';
```

Comment:
Actually, I don't think `lit/static-html` is needed or appropriate in tests. I'll push a commit switching them to the usual `lit`.

Reply:
- [ ] Switch to `lit`